perfdash/Makefile

```diff
@@ -1,7 +1,7 @@
 all: push
 
 # See pod.yaml for the version currently running-- bump this ahead before rebuilding!
-TAG = 0.26
+TAG = 0.27
```

Comment:
Can you please change this to 1.0?

Reply:
Sure - assuming that this is the last required change (for 1.0 version).